python/lsst/pex/config/config.py

```diff
@@ -49,21 +50,23 @@ def _joinNamePath(prefix=None, name=None, index=None):
         name = prefix + "." + name
 
     if index is not None:
-        return "%s[%r]"%(name, index)
+        return "%s[%r]" % (name, index)
     else:
         return name
 
+
 def _autocast(x, dtype):
     """
     If appropriate perform type casting of value x to type dtype,
     otherwise return the original value x
     """
-    if dtype==float and type(x)==int:
+    if dtype == float and type(x) == int:
```

Comment:
I tweaked it and also fixed the `__builtin__` vs `builtins` problem a bit lower down.